djangocms_frontend/ui_plugin_base.py

```diff
@@ -30,12 +30,13 @@ def __str__(self):
         return force_str(super().__str__())
 
     def render(self, context, instance, placeholder):
-        for key, value in instance.config.items():
-            if isinstance(value, dict) and set(value.keys()) == {"pk", "model"}:
-                if key not in instance.__dir__():  # hasattr would return the value in the config dict
-                    setattr(instance.__class__, key, get_related(key))
-        if "instance" not in instance.config and isinstance(instance.config, dict):
-            context.update(instance.config)
+        if hasattr(instance, "config"):
```

Comment:
```suggestion
        if isinstance(instance, AbstractFrontendUIItem):
```
Well spotted: I'd suggest to check more the class of the instance, since the `config` property needs to have the capabilities defined in `AbstractFrontendUIItem`. 

You would also have to import `AbstractFrontendUIItem` from `.models`